legal_db/models.py

```diff
@@ -1,11 +1,135 @@
+from django.contrib.postgres.fields import ArrayField
 from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django_countries.fields import CountryField
+from ordered_model.models import OrderedModel
 
 
-class License(models.Model):
-    code = models.CharField(max_length=12)
-    title = models.CharField(max_length=50)
-    version = models.CharField(max_length=25, blank=True, null=True)
-    notes = models.TextField(blank=True, null=True)
+class BaseModel(models.Model):
+    notes = models.TextField(
+        blank=True, null=True, help_text="Internal notes or description."
+    )
+    created_at = models.DateTimeField(
+        auto_now_add=True, help_text="A timestamp of when the object is first created."
+    )
+    updated_at = models.DateTimeField(
+        auto_now=True,
+        help_text="A timestamp of the last time when the object was modified.",
+    )
+
+    class Meta:
+        abstract = True
+
+
+class LegalResource(BaseModel):
+    license = models.CharField(
+        max_length=50,
+        blank=True,
+        null=True,
+        help_text="The Creative Commons licence associated to the case.",
+    )
+    contributor_name = models.CharField(max_length=120)
+    contributor_email = models.EmailField()
+    contribution_credit_requested = models.BooleanField(
+        help_text="Whether or not show credit to contributor by name on the site."
+    )
+    link = models.URLField(
+        max_length=2000,
+        help_text="Link to the Original Version of Decision (for Case) or the Article (for Scholarship).",
+    )
+    country = CountryField()
+    summary = models.TextField(blank=True, null=True)
+
+    class Status(models.IntegerChoices):
+        UNREVIEWED = 1, _("Unreviewed")
+        REVIEW = 2, _("Review")
+        IN_PROGRESS = 3, _("In progress")
+        PUBLISHED = 4, _("Published")
+        REJECTED = 5, _("Rejected")
+
+    status = models.PositiveSmallIntegerField(
+        choices=Status.choices,
+        default=Status.UNREVIEWED,
+        help_text="The stage of the review process where the resource is.",
+    )
+
+    class Meta:
+        abstract = True
+
+
+class Case(LegalResource):
+    name = models.CharField(max_length=200, blank=True, null=True)
+    case_no = models.CharField(
+        max_length=50, blank=True, null=True, help_text="The number of the case."
+    )
+    jurisdiction = models.CharField(
+        max_length=255, blank=True, null=True, help_text="Jurisdiction Within country."
```

Comment:
Minor nitpick: can we make sure to keep the capitalization of help text consistent in the help text? I think we should use sentence case for help text.

Reply:
I think this applies to other help text as well.

Reply:
Ups, yes, this was a little mistake. Thanks for spotting it. I agree in to keep text consistency.